syntax = "proto3";
option go_package = "RetoFit-App/services/physical_activities_service/internal/app/rest_api/grpc/proto";

package user;

service UserService {
  rpc GetUser (UserRequest) returns (UserResponse);
}

message UserRequest {
  int32 id = 1;
}

message UserResponse {
  int32 id = 1;
  string name = 2;
  string email = 3;
}